util/exec/exec.go

```diff
@@ -36,11 +37,24 @@ func Run(cmd *exec.Cmd) (string, error) {
 func RunWithRedactor(cmd *exec.Cmd, redactor func(text string) string) (string, error) {
 	span := tracing.NewLoggingTracer(log.NewLogrusLogger(logrus.New())).StartSpan(fmt.Sprintf("exec %v", cmd.Args[0]))
 	span.SetBaggageItem("dir", fmt.Sprintf("%v", cmd.Dir))
-	span.SetBaggageItem("args", fmt.Sprintf("%v", cmd.Args))
+
+	var args []string
```

Comment:
@alexmt This is a temporary solution, I am going to fix it in gitops-engine/NewLoggingTracer by adding redactor.